composer.json

```
@@ -61,18 +66,19 @@
     }
   ],
   "require-dev": {
+    "php": "^7.4 || ^8.0",
     "alleyinteractive/alley-coding-standards": "^0.3.0",
+    "alleyinteractive/pest-plugin-wordpress": "dev-main",
     "alleyinteractive/searchpress": "^0.4.1",
     "automattic/vip-go-mu-plugins-built": "dev-master",
     "composer/installers": "^1.0",
-    "phpunit/phpunit": "7.5.*",
     "wpackagist-plugin/elasticpress": "^4.2",
     "yoast/phpunit-polyfills": "^1.0"
   },
   "scripts": {
     "phpcbf": "phpcbf .",
     "phpcs": "phpcs .",
-    "phpunit": "phpunit",
+    "phpunit": "pest",
     "setup": [
```

For ease of use, it might be nice to also support `composer run test`:

```suggestion
    "phpunit": "@test",
    "test": "pest",
    "setup": [
```

This will reroute `composer run phpunit` to `composer run test`. I'm trying to get most projects to support either `npm run test` or `composer run test`.